### ISSUE TITLE
TS Lint failure with Axios 0.19

### ISSUE THREAD

**Issue by H3llK33p3r** [vojtechhabarta/typescript-generator#379]
Hi, the axios library has changed the AxiosRequestConfig interface from 

```
export interface AxiosRequestConfig {
  url?: string;
  method?: string;
```

to 

```
export interface AxiosRequestConfig {
  url?: string;
  method?: Method;
```

So client generation do generate an error when using tslint with axios 0.19.* : 
![image](https://user-images.githubusercontent.com/4510572/60250592-3d8bd980-98c7-11e9-9124-16eed075121d.png)

BR.

**Comment by vojtechhabarta** [comment]
Hi, I committed change which casts `string` parameter to correct type. I also upgraded axios version in generated package.json to 0.19.0.

**Comment by vojtechhabarta** [comment]
Released in [v2.16.538](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.16.538).
